ukrainian/use-the-force/Darth/darth-act/end.md

```diff
@@ -0,0 +1,4 @@
+Він схопив вас за горло за допомогою сили 
+та ви знепритомніли.
+
+[Прокинутись](https://github.com/mmisesin/create-your-own-adventure/blob/mmisesin_update/ukrainian/marshmallow.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.